audit/audit_base.hpp

```diff
@@ -32,6 +32,8 @@ namespace bzn
     const std::string COMMIT_METRIC_NAME = "stats.commit_heard";
     const std::string NEW_LEADER_METRIC_NAME = "stats.new_leader_heard";
 
```

Comment:
We will want to better organize this as:

com.bluzelle.swarm.UUID.node.UUID.stats.commit_heard
com.bluzelle.swarm.UUID.node.UUID.stats.new_leader_heard